base/checked.jl

```diff
@@ -295,11 +307,8 @@ Calculates `div(x,y)`, checking for overflow errors where applicable.
 
 The overflow protection may impose a perceptible performance penalty.
 """
-function checked_div end
-
 # Base.div already checks; nothing to do here
```

Comment:
not sure if it matters, but there probably shouldn't be a comment between a docstring and the function it documents?